extern/algebra-plugins/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building Algebra Plugins as part of the Detray project" )
+
+# Declare where to get Algebra Plugins from.
+FetchContent_Declare( AlgebraPlugins
+   URL "https://github.com/acts-project/algebra-plugins/archive/refs/tags/0.1.0.tar.gz"
```

Comment:
I only realised now that this is not quite right... :frowning: Let me rename the first release to `v0.1.0` instead. Since that's the format that VecMem uses as well. (And is what's recommended by GitHub in general...)

Reply:
Done.